presto-docs/src/main/sphinx/release/release-0.231.rst

```diff
@@ -0,0 +1,52 @@
+=============
+Release 0.231
+=============
+
+General Changes
+_______________
+* Fix failures caused by invalid plans for queries with repeated lambda expressions in the order by clause.
+* Fix accounting of raw input bytes in ScanFilterAndProjectOperator.
+* Fix an issue where server fails to start when two function namespace managers of the same type are specified.
+* Fix SQL function compilation error when input parameters contain lambda.
+* Fix IS DISTINCT FROM for long decimals.
+* Improve error handling for geometry deserialization edge cases.
+* Improve read-path to better estimate the size of row. Output (sum) string stat even if min/max values are too long.
```

Comment:
I also don't quite understand this sentence. From #13879 it seems that this is about improving stats. @wenleix could you help reword this? Otherwise I think the commit message itself is better than this: `Keep sum stat for string when min/max stat is too long in ORC writer.`